region/tourian/main.json

```diff
@@ -203,8 +240,19 @@
           "to": [
             {
               "id": 1,
-              "requires": null,
-              "unlock": [ "canKillMetroids" ]
+              "requires": [
+                {"or": [
+                  "SpaceJump",
+                  {"cost": [
+                    {"acidFrames": 60}
+                  ]}
+                ]},
+                {"or": [
```

Comment:
Does this not make the room entirely out of logic with PBs? You can unlock the door but you can't reach it.